modules/claims_api/spec/requests/v1/disability_compensation_request_spec.rb

```diff
@@ -1402,6 +1402,34 @@ def obj.class
           end
         end
       end
+
+      context "when either 'currentlyHomeless' nor 'homelessnessRisk' is provided" do
```

Comment:
Should this be `or` instead of `nor`?

Reply:
and `are` instead of `is`?

Reply:
`nor` should definitely be `or`. Second one could go either way, [it appears](https://www.quora.com/Which-one-is-correct-either-is-correct-or-either-are-correct?share=1)